### PR TITLE
4292: Implemented table for comparing public/preview section values

### PR DESCRIPTION
**Pull request by CallumBradburyBJSS**

### CODE REVIEW

File: tests/NHSD.BuyingCatalogue.API.IntegrationTests/Features/Public/Display Marketing Page Public Solution Description Section.3576.feature [hunk 1]
```diff
@@ -4,37 +4,31 @@ Feature: Display Marketing Page Public Solution Description Section
     So that I can ensure the information is correct
```
Comment (sam-roberts-bjss) [inline tests/NHSD.BuyingCatalogue.API.IntegrationTests/Features/Public/Display Marketing Page Public Solution Description Section.3576.feature]: Great spot on this! 💯

File: tests/NHSD.BuyingCatalogue.API.IntegrationTests/Features/Preview/Display Marketing Page Preview Solution Section.3507.feature [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+@3507
+Scenario: 2. Solution section is presented where the solution isFoundation is false
+    When a GET request is made for solution preview Sln2
+    Then a successful response is returned
+    And the string value of element supplierName is Supplier 2
+    And the string value of element isFoundation is False
+    And the last updated date in the solution is 15/11/2019
+
+@3507
+Scenario: 3. Solution is not linked to Framwork Solution
```
Comment (sam-roberts-bjss) [inline tests/NHSD.BuyingCatalogue.API.IntegrationTests/Features/Preview/Display Marketing Page Preview Solution Section.3507.feature]: Framework?

File: tests/NHSD.BuyingCatalogue.API.IntegrationTests/Features/Preview/Display Marketing Page Preview Browser Based Section.3322.feature [hunk 3]
```diff
@@ -4,72 +4,29 @@ Feature: Display Marketing Page Preview Browser Based Section
...
-        | Edge          |
-    And the solution client-application-types section contains mobile responsive with value No
-    And the solution client-application-types section contains hardware-requirements with value New Hardware
-    And the solution client-application-types section contains additional-information with value Some Additional Info
-    And the solution client-application-types section contains minimum-connection-speed with value 1GBps
-    And the solution client-application-types section contains minimum-desktop-resolution with value 1x1
-    And the solution client-application-types section contains browser-mobile-first with value Yes
-
-@3322
-Scenario:2. Get Solution Preview contains client application types browser based mobile responsive answer
```
[the remark this answers is not in the document]
Comment (CallumBradburyBJSS) [reply tests/NHSD.BuyingCatalogue.API.IntegrationTests/Features/Preview/Display Marketing Page Preview Browser Based Section.3322.feature]: They were removed for consistency, as mid-development they were deemed redundant. There will be extra tests added to cover the genuine use cases of SolutionDetail not existing etc, once the 2nd half of the change is done, but these were verifying the exact same thing as Scenario 1

File: tests/NHSD.BuyingCatalogue.API.IntegrationTests/Features/Preview/Display Marketing Page Preview Solution Section.3507.feature (hunk 2 above)
Comment (CallumBradburyBJSS) [reply tests/NHSD.BuyingCatalogue.API.IntegrationTests/Features/Preview/Display Marketing Page Preview Solution Section.3507.feature]: This is meant to be Foundation, I assume? these were a carbon copy of the Public tests, as they weren't being tested in Preview. I'll make the change in both!

**Pull request merged**